modules/mariadb/src/main/java/org/testcontainers/containers/MariaDBContainer.java

```diff
@@ -17,8 +17,10 @@ public class MariaDBContainer<SELF extends MariaDBContainer<SELF>> extends JdbcD
 
     private static final DockerImageName DEFAULT_IMAGE_NAME = DockerImageName.parse("mariadb");
 
+    private static final DockerImageName FOUNDATION_IMAGE_NAME = DockerImageName.parse("quay.io/repository/mariadb-foundation/mariadb-devel");
+
     @Deprecated
-    public static final String DEFAULT_TAG = "10.3.6";
+    public static final String DEFAULT_TAG = "10.11.10";
```

Comment:
default constructor is deprecated and for those using it will bring a new version, which is unexpected. User must declare the version they want to use